README.md

```diff
@@ -18,28 +18,99 @@ In order to use the connector, you need a `SqsAsyncClient`. Refer to the [AWS SD
 
 ### Publish messages
 
-Use `SqsPublisher.send` to publish messages to a queue.
+Use `Producer.make` to instantiate an instance of [Producer](src/main/scala/zio/sqs/producer/Producer.scala) trait that can be used to publish objects of type `T` to the queue.
```

Comment:
Do we keep the old way of sending messages? You deleted the docs but kept the code.